test/abi/abi.inputParser.js

```diff
@@ -1,7 +1,7 @@
-var assert = require('assert');
+var assert    = require('assert');
 var BigNumber = require('bignumber.js');
-var abi = require('../lib/solidity/abi.js');
-var clone = function (object) { return JSON.parse(JSON.stringify(object)); };
+var abi       = require('../../lib/solidity/abi.js');
+var clone     = function (object) { return JSON.parse(JSON.stringify(object)); };
```

Comment:
Sure, I knew we would end up talking about it :smile: 
I already tuned up my editor profile to match what I could see from the existing code.
I personally find those aligment a bit sexier but that´s no must, so I leave it up to you if you care.